mlir/include/mlir/Dialect/OpenMP/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ mlir_tablegen(OmpCommon.td --gen-directive-decl --directives-dialect=OpenMP)
 add_public_tablegen_target(omp_common_td)
 
 set(LLVM_TARGET_DEFINITIONS OpenMPOps.td)
+mlir_tablegen(no-output -verify-openmp-ops)
```

Comment:
Hmm, will this generate a file called `no-output`, which will further me included into the `MlirOpenMPOpsIncGen` target below?

Reply:
Thank you for the quick review! I suppose it might do, though if that's the case it will always be empty. I'm not very familiar with how this infrastructure works, so I don't know if that could cause an issue. After a quick search I can't find a definition for `add_public_tablegen_target` to check this either 🤔.

Reply:
I made some changes here that basically remove the empty "no-output" files from the build directory and also prevent it from being added as a dependency. It's admittedly a bit of a hack, but I don't know if it's worth the effort to refactor the `mlir_tablegen` and `tablegen` CMake functions to allow non-generators like this one, since it seems like something that won't be useful that often.

Reply:
Thanks for investigating! It's not a big issue if the file is still present, but better avoid it in dependencies. In case you get complains about windows builds where paths are different, feel free to drop the file removal and only keep the variable manipulation.